tests/Feature/Rendering/JsonApiTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Tests\Feature\Rendering;
+
+use Apitizer\Rendering\JsonApiRenderer;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Collection;
+use Tests\Feature\Models\User;
+use Tests\Feature\TestCase;
+use Tests\Support\Builders\UserBuilder;
+
+/**
+ * The query builder can be used JUST to render data, rather than also fetching
+ * it. That's what this class tests.
+ */
+class JsonApiTest extends TestCase
+{
+    /** @test */
+    public function it_renders_existing_eloquent_model_to_jsonapi_format()
```

Comment:
I would suggest
`public function we_can_use_the_json_api_renderer_to_format_the_builder_response_in_json_api_format()`

Mainly because I am looking at what this PR does from the tests. Same goes for the one below.